hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -182,6 +183,13 @@ public boolean commitStats(String instantTime, List<HoodieWriteStat> stats, Opti
     try {
       preCommit(instantTime, metadata);
       commit(table, commitActionType, instantTime, metadata, stats);
+      if (extraMetadata.isPresent() && extraMetadata.get().containsKey("_hoodie.zorder.enable")) {
```

Comment:
can define constant in `HoodieClusteringConfig`?

Reply:
this prameter is internal used. i will try to optimize it